sum_up_zero.cpp

```diff
@@ -91,15 +92,23 @@ int main()
 	int n=6;
 	std::vector<int> s1(s,s+n); 
 	std::set<std::vector<int> > ans1=three_sum(s1,6,0);
+	cout<<"{-1, 0, 1, 2, -1, -4} =";
 	print_match(ans1);
 	//should print (-1  1  0  ) (-1  2  -1  )
 
 	int sb[]={1, 0, 1, 2, 1, 4};
 	n=6;
 	std::vector<int> s2(sb,sb+n);
+	cout<<"{1, 0, 1, 2, 1, 4} =";
     print_match(three_sum(s2,6,0));	
 	//sould print nothing or ()
```

Comment:
It would probably be good to comment what you are testing in the test like
// no triplets in array...
Or something like that